testing/frr/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Maintainer: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0.1
+pkgrel=0
+pkgdesc="FRRouting is a fork of quagga"
+pkgusers="frr"
+pkggroups="frr"
+url="https://frrouting.org/"
+arch="x86_64"
+license="GPL-2.0-only and LGPL-2.1-only"
+depends="json-c c-ares iproute2 python2 py-ipaddr bash"
```

Comment:
`json-c` and `c-ares` should not be needed in `depends` as abuild will automatically add shared libraries to depends.

Does `frr` support python3 instead of python2? I want avoid adding new packages that depends on python2 it its not necessary.

`py-ipaddr` should be versioned. eg. `py2-ipaddr` or `py3-ipaddr`.

Reply:
I'm not sure about python2 vs python3, right now, it seems like other distro packages are using python2.  The python code needs to be cleaned up upstream (using console scripts, modules, etc) I was hoping to do the python2 and python3 testing once I get those changes upstream.  I do want to go w/ python3.  frr-reload is also due to be replaced upstream, so hoping not to put too much work into that...

Will test/fix json-c c-ares out of depends and py-ipaddr...

Dependencies were cut and paste from other distro packages and docs, @tbk also pointed that out and I trimmed it down, but it looks like I should come at it from the other side and just add in dependencies one by one...